rebasehelper/tags.py

```diff
@@ -57,6 +57,27 @@ def __getitem__(self, index):
     def __len__(self):
         return len(self.items)
 
+    @classmethod
+    def _map_sections_to_parsed(cls, raw_content: SpecContent, parsed_content: SpecContent) -> Dict[int, int]:
+        """Creates a mapping of raw sections to parsed sections.
+
+        Returns:
+            A dict where the keys are indexes of sections in raw content and values are their
+            counterparts in parsed content. Value is set to -1 if the section is not in
+            parsed SpecContent.
+        """
+        result: Dict[int, int] = {}
+        parsed = 0
+        for index, (section, _) in enumerate(raw_content.sections):
+            if parsed >= len(parsed_content.sections) or parsed_content.sections[parsed][0].lower() != section.lower():
```

Comment:
If the first condition is true, what's the point in continuing the iteration?

Reply:
We have to go through the remaining sections to set its parsed counteparts as undefined (`-1`). If I added a break there, then the mapping wouldn't contain all of the raw section, would it?

Reply:
Ok.